### DISCUSSION TITLE
Trying to setup up Forwarding to my RPi 4 from a windows desktop software screen capture instance.

### DISCUSSION THREAD

**Discussion by Jenematalia** [awawa-dev/HyperHDR#286] [Q&A] [answered]
I have enabled Forwarding in the Remote Control tab, but in Network Services tab there is no option for setting up the Forwarder Service. Not sure if I'm just blind or what. Where do I find the options to point this desktop instance to my raspberry pi 4 hooked up to my TV.
<img width="983" alt="Screenshot 2022-06-09 064106" src="https://user-images.githubusercontent.com/102889065/172861334-347c2da0-1fcd-4fdd-a6f2-2d3d3a17e3cb.png">

**Comment by awawa-dev** [comment]
Enable advanced user settings

**Comment by Jenematalia** [comment]
Alright I did that and now I get the forwarder tabs, however I'm not sure I'm doing this correctly. I have my desktop pointing at my rpi 4's IP and the ports match, I'm getting a live video preview on the desktop so I know the screen capturing is working; the raspberry is not showing the desktop as an option of Source. Am referencing this discussion and have it set up just like this, same port and everything: https://github.com/awawa-dev/HyperHDR/discussions/149 I appreciate your help btw.

**Comment by awawa-dev** [comment, marked as answer]
Verify logs on both units. BTW new v18 UDPRaw colors forwarding is much faster than sending raw uncompressed images to Rpi. It's explained in the network component how it works.

**Reply by Jenematalia** [reply]
So I got it to run and forward it all to the rpi 4 but it is extremely slow. I have v16 installed on the rpi 4, will it help if I update it? I have the LED Controller type set to: udpraw on the desktop sender. If I don't have to update the pi that would be desired since I'm not super fluent in updating OTA.

**Reply by awawa-dev** [reply]
Probably your network is saturated sending full images and upgrade won't help for flatbuffers with current settings. How high is resolution?

**Reply by Jenematalia** [reply]
Literally didn't touch a thing and walked away for a couple minutes and now it's forwarding at normal speed. I have the resolution set to the lowest 196 and 30 fps. Not sure what fixed anything but it works great now. Thanks for your help, you're the best. Quick question though, how do you activate a certain color calibration profile? I see that I can create multiple profiles in the image processing tab but how do I switch between them?

**Reply by Jenematalia** [reply]
or do they cascade on top of each other as you make more?

**Reply by awawa-dev** [reply]
You can create only one also as a cascade of child sub-segments. So you can't switch them. But you can manipulate processing using JSON API.
